scripts/skills/actives/rf_pummel_skill.nut

```diff
@@ -0,0 +1,129 @@
+		];
+		this.m.Order = ::Const.SkillOrder.Any;
+		this.m.ActionPointCost = 6;
+		this.m.FatigueCost = 20;
+		this.m.AIBehaviorID = ::Const.AI.Behavior.ID.LineBreaker;
+	}
+
+	function getTooltip()
+	{
+		local ret = this.skill.getDefaultUtilityTooltip();
+
+		ret.push({
+			id = 10,
+			type = "text",
+			icon = "ui/icons/special.png",
+			text = ::Reforged.Mod.Tooltips.parseString("Has a " + ::MSU.Text.colorGreen("100%") + " chance to [stagger|Skill+staggered_effect] the target")
+		});
+
+		ret.push({
+			id = 11,
+			type = "text",
+			icon = "ui/icons/special.png",
+			text = "If the attack is successful, the target is pushed back a tile and you move into their position"
+		});
+
+		local attack = this.getValidAttack();
+		if (attack == null)
+		{
+			ret.push({
+				id = 12,
+				type = "text",
+				icon = "ui/icons/warning.png",
+				text = ::MSU.Text.colorRed("Requires an attack from a two-handed hammer")
+			});
+		}
+		else
+		{
+			ret.push({
+				id = 12,
+				type = "text",
+				icon = "ui/icons/special.png",
+				text = ::Reforged.Mod.Tooltips.parseString(format("Perform a free [%s|Skill+%s] on the target", attack.getName(), split(::IO.scriptFilenameByHash(attack.ClassNameHash), "/").top()))
+			});
+		}
+
+		if (this.getContainer().getActor().getCurrentProperties().IsRooted)
+		{
+			ret.push({
+				id = 13,
+				type = "text",
+				icon = "ui/icons/warning.png",
+				text = ::MSU.Text.colorRed("Cannot be used while Rooted")
+			});
+		}
+
+		return ret;
+	}
+
+	function isUsable()
+	{
+		return this.line_breaker.isUsable() && !this.getContainer().getActor().getCurrentProperties().IsRooted && this.getValidAttack() != null;
+	}
+
+	function onUse( _user, _targetTile )
+	{
+		local targetEntity = _targetTile.getEntity();
+
+		local aoo = this.getValidAttack();
+		local overlay = aoo.m.Overlay;
+		aoo.m.Overlay = "";
+		local success = aoo.useForFree(_targetTile);
+		aoo.m.Overlay = overlay;
+
+		if (success)
+		{
+			if (targetEntity.isAlive())
+			{
+				targetEntity.getSkills().add(::new("scripts/skills/effects/staggered_effect"));
+				if (!_user.isHiddenToPlayer() && _targetTile.IsVisibleForPlayer)
+					::Tactical.EventLog.log(::Const.UI.getColorizedEntityName(_user) + " has pummeled and staggered " + ::Const.UI.getColorizedEntityName(targetEntity));
+			}
+
+			local tag = {
+				User = _user,
+				TargetTile = _targetTile
+				TargetEntity = targetEntity
+			}
+			// Doing this.line_breaker.onUse here directly doesn't work properly:
+			// The targetEntity gets knocked back by Linebreaker, but the attacker does not move to the target tile.
+			// Using scheduleEvent even with a delay of 1 is enough to make it work properly. No idea why. -- Midas
+			::Time.scheduleEvent(::TimeUnit.Virtual, 1, this.onPushThrough.bindenv(this), tag);
+		}
+
+		return success;
+	}
+
+	function onPushThrough( _tag )
+	{
+		if (_tag.TargetEntity.isAlive())
+			this.line_breaker.onUse(_tag.User, _tag.TargetTile);
```

Comment:
Linebreaker builds up 10 Fatigue on the target, this minor effect is not mentioned in the perks/skills description.

Reply:
Should we remove this 10 Fatigue damage from Pummel or keep it and add it to the tooltip?

Reply:
I think we should keep it for now, it adds some nice flavor to the skill.